files/image_config/platform/rc.local

```diff
@@ -340,4 +340,10 @@ if [ -f $FIRST_BOOT_FILE ]; then
     firsttime_exit
 fi
 
+# Copy the fsck log into syslog
```

Comment:
>Copy [](start = 2, length = 4)

Why copy to syslog?

Reply:
syslog being the unified way of seeing system messages, we thought that its was more appropriate to hold the fsck log messages instead of a compressed file under the /var/log directory